cmd/node-termination-handler.go

```diff
@@ -22,33 +22,47 @@ import (
 
 	"github.com/aws/aws-node-termination-handler/pkg/config"
 	"github.com/aws/aws-node-termination-handler/pkg/drainer"
-	"github.com/aws/aws-node-termination-handler/pkg/ec2metadata"
+	"github.com/aws/aws-node-termination-handler/pkg/drainevent"
 )
 
 func main() {
 	setupSignalHandler()
 	nthConfig := config.ParseCliArgs()
 	drainer.InitDrainer(nthConfig)
+	drainEventStore := drainevent.Store{
+		NthConfig: &nthConfig,
+	}
+
+	log.Println("Kubernetes AWS Node Termination Handler has started successfully!")
+	drainChan := make(chan drainevent.DrainEvent)
```

Comment:
technically you should `close()` this channel. potentially put the chan definition in the module scope and `close()` it in your signal handler.

Reply:
I thought about this with mrbecker a bit and there really isn't a need for the signal handler we have now anyways... so I refactored the code to exit normally and then just used defers.

Reply:
Cool with me! :)